components/automate-cli/pkg/verifyserver/services/s3configservice/s3configservice.go

```diff
@@ -30,7 +32,14 @@ func NewS3ConfigService(logger logger.Logger, awsUtils awsutils.AwsUtils) IS3Con
 
 func (ss *S3ConfigService) GetS3Connection(req *models.S3ConfigRequest) *models.Checks {
 	ss.Req = req
-	sess, err := ss.AwsConnection(ss.Req.Endpoint, ss.Req.AccessKey, ss.Req.SecretKey, "")
+
+	// aws-sdk is expecting the endpoint url of S3 bucket to be in https://s3.REGION.amazonaws.com pattern
+	// https://s3.amazonaws.com by default belongs to us-east-1 region and if the bucket is in someother region, the check will not pass
+	// So manipulating the endpoint URL pattern for S3 (ONLY FOR AWS S3)
+	if strings.Contains(ss.Req.Endpoint, "://s3.amazonaws.com") {
+		ss.Req.Endpoint = strings.Replace(ss.Req.Endpoint, "s3.amazonaws.com", "s3."+ss.Req.Region+".amazonaws.com", 1)
```

Comment:
@iamazzeez & @ArvinthC3000 
Whenever req has an S3 endpoint, is it expected to have a region as well?
cc: @punitmundra